metadata-ingestion/tests/unit/test_packages.py

```diff
@@ -0,0 +1,10 @@
+import setuptools
+
+def test_package_list_match_inits():
+    where = "./src"
+    package_list = set(setuptools.find_packages(where))
+    namespace_packages = set(setuptools.find_namespace_packages(where))
+    diff1 = package_list.difference(namespace_packages)
```

Comment:
Just assert that the two sets are equal to each other

Also maybe add a comment that diffs are likely due to missing init files

Reply:
done